docs/source/datasets/index.rst

```diff
@@ -281,7 +281,7 @@ The following columns are included in this dataset:
      - Rounded down to an integer.
    * - Date of birth
      - :code:`date_of_birth`
-     - Formatted as YYYY-MM-DD.
+     - Formatted as MMDDYYYY.
```

Comment:
These date formatting changes shouldn't be in this PR, as they are covered in #216.

Reply:
Ooh okay idk why I got nervous there would be a merge conflict so I thought this would be proactive but I can undo them..